### PR TITLE
Add endpoint for deleting searches

### PR DESCRIPTION
**Pull request by fabubaker**

### CODE REVIEW

File: .insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+_id: req_84c3c10ae54b4947b1bfae581f94db27
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1660088304634
+created: 1660087332898
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/searches/{% response
```
Comment (hammady) [inline .insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml:6]: How does this look like? I suggest adding a screenshot for every PR.

Comment (fabubaker) [reply .insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml:6]: Like this:

image

Comment (hammady) [reply .insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml:6]: So this basically deletes the same search that was created in the preceding endpoint?

**Review by hammady — APPROVED** [review]

File: .insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml (hunk 1 above)
Comment (fabubaker) [reply .insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml:6]: Yeah, pretty much. Isn't this preferred? The alternative would be to supply an id manually.

Comment (hammady) [reply .insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml:6]: No problem.

**Pull request merged**